taxonomies/categories.txt

```diff
@@ -68,6 +68,7 @@ fr:Gressins artisanaux
 hu:Kézzel készített termékek
 it:Grissini artigianali, Grissini fatti a mano
 nl:Handgemaakte broodstengels
+nl_be:Handgemaakte broodstengels
```

Comment:
Currently we do not support language variants, so only nl: is used, nl_be: will not have an effect (it's ok to add it though).

Reply:
> Currently we do not support language variants, so only nl: is used, nl_be: will not have an effect (it's ok to add it though).

Ohh, I didn't know! Thanks for letting me know!